WalletWasabi/Affiliation/IAffiliateServerApi.cs

```diff
@@ -0,0 +1,12 @@
+using System.Threading;
+using System.Threading.Tasks;
+using WalletWasabi.Affiliation.Models;
+
+namespace WalletWasabi.Affiliation;
+
+public interface IAffiliateServerApi
```

Comment:
I have no problem removing it. I agree it is not necessary but it clearly documents the API of the affiliate server. What do you think?

Reply:
We make an effort to avoid interfaces as much as possible. Many times we were tempted to use them for unit testing or breaking some dependencies but we realize it is possible (and good) to don't have them. We have never used interfaces for documenting, i know that's a common and desirable thing in many languages (i am thinking in ocaml) but we don't do it and i think we don't need it neither.

Reply:
Ok. Done in https://github.com/zkSNACKs/WalletWasabi/pull/9769/commits/6cbfa6ec17b955f718aa2ba8efe18b51bd16c4f3.